gatsby/src/pages/drugs/{BnfDrug.slug}/medicinal-forms.tsx

```diff
@@ -89,9 +197,19 @@ export const query = graphql`
 				medicinalForms {
 					form
 					slug
+					electolytes
+					excipients
 					preps {
 						...FullPrep
 					}
+					cautionaryAndAdvisoryLabels {
+						label {
+							number
+							englishRecommendation
+							welshRecommendation
+						}
+						additionalNotes
```

Comment:
Some places this is called `additionalInfo` and some places it's `additionalNotes` but this will change with the new feed tomorrow to, I think, `qualifier`